plugins/QnA/class.qna.plugin.php

```diff
@@ -925,17 +925,18 @@ public function discussionsController_unanswered_create($sender, $args) {
      */
     public function discussionsController_beforeBuildPager_handler($sender, $args) {
         if (Gdn::controller()->RequestMethod == 'unanswered') {
-            $count = $this->getUnansweredCount();
+            $count = $this->getUnansweredCount(true);
             $sender->setData('CountDiscussions', $count);
         }
     }
 
     /**
      * Return the number of unanswered questions.
      *
+     * @param bool $pager Whether this count affects the pager or not.
      * @return int
      */
-    public function getUnansweredCount() {
+    public function getUnansweredCount($pager = false) {
         // TODO: Dekludge this when category permissions are refactored (tburry).
         $cacheKey = Gdn::request()->webRoot().'/QnA-UnansweredCount';
         $questionCount = Gdn::cache()->get($cacheKey);
```

Comment:
If we add extra parameter here, we should either differ `$cacheKey` for 2 different values or bypass caching for one of them.

I would prefer probably to go with 1st attempt. Implement 2 `$cacheKey` variants.